### ISSUE TITLE
[Bug] Test Rush Upgrade

### ISSUE THREAD

**Issue by Biangkerok32** [shreyashsaitwal/rush-cli#23]
**Test Rush Upgrade**

I dont know is bug or not.
im try new rush version 1.2.0

and try enter command "rush upgrade"

**Log**

```
PS C:\Users\kangris> rush upgrade
                      __
     _______  _______/ /_
    / ___/ / / / ___/ __ \
   / /  / /_/ (__  / / / /
  /_/   \__,_/____/_/ /_/

info Fetching data...
Unhandled exception:
GitHub Error: Bad credentials
#0      RepositoriesService.getContents.<anonymous closure> (package:github/src/common/repos_service.dart:573)
#1      GitHub.requestJson (package:github/src/common/github.dart:280)
<asynchronous suspension>
#2      UpgradeUtils.fetchContents (package:rush_cli/commands/upgrade_command/helpers/upgrade_utils.dart:18)
<asynchronous suspension>
#3      UpgradeCommand.run (package:rush_cli/commands/upgrade_command/upgrade_command.dart:36)
<asynchronous suspension>
#4      CommandRunner.runCommand (package:args/command_runner.dart:196)
<asynchronous suspension>
PS C:\Users\kangris>

```

**Desktop (please complete the following information):**
 - OS: [Windows]
 - Version [1.0.0]

**Comment by shreyashsaitwal** [comment]
Thanks for reporting this. I'm already aware of that. It's because, unfortunately, GitHub has revoked my personal access token, that is required for downloading files, because of a security breach.

I've contacted GitHub regarding this, but I doubt even they would be able to do anything in this case. Let's see what happens.

**Comment by shreyashsaitwal** [comment]
Fixed by [v1.2.1](https://github.com/shreyashsaitwal/rush-cli/releases/tag/v1.2.1)
